src/app/Pages/TransactionHistory.tsx

```diff
@@ -17,11 +21,47 @@ export default function TransactionHistory(): JSX.Element {
   }
 
   return (
-    <div>
+    <Container>
       <TransactionOverview
         transactions={transactions}
         deleteTransaction={handleDeleteTransaction}
       />
-    </div>
+      <Link to="/">
+        <StyledButton>
+          <Icon>
+            <ArrowIconRight />
+          </Icon>
+          <ButtonText>Startseite</ButtonText>
+        </StyledButton>
+      </Link>
+    </Container>
   )
 }
+
+const Container = styled.div`
+  display: grid;
+  grid-template-rows: auto 60px;
+`
+
+const StyledButton = styled(Button)`
```

Comment:
Is there a better name, maybe the function of this button? What does the button do exactly?

Reply:
very nice..thanks